internal/subshell/subshell.go

```diff
@@ -204,7 +211,7 @@ func DetectShell(cfg sscommon.Configurable) (string, string) {
 	}
 
 	isKnownShell := false
-	for _, ssName := range []string{bash.Name, cmd.Name, fish.Name, tcsh.Name, zsh.Name} {
+	for _, ssName := range []string{bash.Name, cmd.Name, fish.Name, tcsh.Name, zsh.Name, pwsh.Name} {
```

Comment:
You can remove the check for powershell on line 223 now since we officially recognize it.

Reply:
Thanks for catching!